src/Libraries/CoreNodes/FileSystem.cs

```diff
@@ -98,12 +98,28 @@ public static void DeleteFile(string path)
         ///     Copies a file.
         /// </summary>
         /// <param name="file">File object to copy</param>
-        /// <param name="destinationPath">String representation of destination path</param>
+        /// <param name="destinationPath">String representation of destination file path</param>
         /// <param name="overwrite">Toggle to overwrite existing files</param>
-        /// <returns name="void">Node performs a task, doesn’t produce an output </returns>
-        public static void CopyFile(FileInfo file, string destinationPath, bool overwrite = false)
+        /// <returns name="bool">Node performs a task, return true of copy action succeed.</returns>
+        public static bool CopyFile(FileInfo file, string destinationPath, bool overwrite = false)
```

Comment:
@QilongTang this change likely means that any c# packages which called this function from a ZT node will need to be recompiled.

That probably is a very small number.

Reply:
Thanks for the reminder! @mjkkirschner